app/src/main/java/org/fcitx/fcitx5/android/data/theme/CustomThemeSerializer.kt

```diff
@@ -80,8 +99,8 @@ object CustomThemeSerializer : JsonTransformingSerializer<Theme.Custom>(Theme.Cu
 
     private const val VERSION = "version"
 
-    private const val CURRENT_VERSION = "2.0"
-    private const val FALLBACK_VERSION = "1.0"
+    private const val CURRENT_VERSION = "2.1"
+    private const val FALLBACK_VERSION = "2.0"
```

Comment:
```suggestion
    private const val FALLBACK_VERSION = "1.0"
```

`FALLBACK_VERSION` should not be changed, it's the "fallback" version when importing themes without "version" key